app/models/app.rb

```diff
@@ -126,6 +127,15 @@ def bitbucket_repo?
     bitbucket_repo.present?
   end
 
+  def build_custom_backtrace_url(file, line)
+    format(custom_backtrace_url, branch: repo_branch, file: file, line: line,
+                                 ebranch: CGI.escape(repo_branch), efile: CGI.escape(file))
```

Comment:
Can you explain the need for both escaped and unescaped names? It seems surprising that the escaped branch alone is insufficient.

Reply:
Wel in my test-cases I've only used/needed the unescaped values. I've added the ebranch and efile names, to support other url's. 
CGI.esacpes also escapes the '/' token. Which doesn't work for the gitea and gilab pathnames and branchnames. 

If you want I can remove the ebranch and efile fields. (They can be added later if somebody runs into problems with other git repos)

Reply:
Seems useful enough to keep. I was just curious.